internal/cmd/server.go

```diff
@@ -46,6 +47,9 @@ func newServerCmd() *cobra.Command {
 
 			options.DBEncryptionKey = dbEncryptionKey
 
+			// override default strcase.ToLowerCamel behaviour
+			strcase.ConfigureAcronym("enable-ui", "enableUI")
```

Comment:
Doesn't this need to be called before `parseOptions`? I thought that's where `strcase` was used.